docs/com.redhat.viaq-openshift-operations.asciidoc

```diff
@@ -197,6 +203,16 @@ to correlate data with the data stored in Elasticsearch, this field
 will give you the exact document corresponding to the record.
 
 
+==== viaq_index_name
+
+type: keyword
+
+example: project.my-cool-project-in-lab04.748e92c2-70d7-11e9-b387-000d3af2d83b.2019.05.09
```

Comment:
@lukas-vlcek please change to match an index in the new data model

Reply:
@jcantrill We shall probably point to https://github.com/openshift/enhancements/blob/master/enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md#data-model instead, WDYT?

Reply:
@jcantrill To address this we need to cut a new release of the model. PTAL https://github.com/ViaQ/elasticsearch-templates/pull/118